balloc/balloc.c

```diff
@@ -628,60 +628,63 @@ struct balloc_group_write_cfg {
 struct balloc_groups_write_cfg {
 	struct balloc_group_write_cfg *bgs_bgc;
 	struct m0_balloc              *bgs_bal;
-	m0_bcount_t                    bgs_current;
 	m0_bcount_t                    bgs_max;
 	int                            bgs_rc;
 	struct m0_mutex                bgs_lock;
 };
 
-static void balloc_group_write_next(struct m0_be_tx_bulk  *tb,
-                                    struct m0_be_op       *op,
-                                    void                  *datum,
-                                    void                 **user)
+static void
+balloc_group_write_credit(struct m0_balloc               *bal,
```

Comment:
Please add some comments over each function so that it helps code reading.

Reply:
@shashank-parulekar: if this is about balloc then this was just a refactoring. If this is about tx_bulk - will do.